sql/auto_install.sql

```diff
@@ -28,17 +81,23 @@ SET FOREIGN_KEY_CHECKS=1;
 -- *
 -- *******************************************************/
 CREATE TABLE `civicrm_civiawards_award_detail` (
```

Comment:
@reneolivo , can you clean up this file a bit so it looks better. The Civicrm comments above can be removed, we don't need it. The SQL queries in this file also does not have any formatting and are scattered all over the place, can you format it similar to how it was before the change?  Thanks.